appcues/src/main/java/com/appcues/ui/presentation/ViewPresenter.kt

```diff
@@ -47,6 +47,10 @@ internal abstract class ViewPresenter(
         override fun onActivityChanged(activity: Activity) {
             viewModel?.onActivityChanged()
         }
+
+        override fun onConfigurationChanged(activity: Activity) {
+            viewModel?.onConfigurationChanged()
```

Comment:
the `ViewPresenter` already has the `experienceRenderer` and the `renderContext`, do we need to send this through the `viewModel` at all? thinking we could short circuit that and just do 

```kotlin
coroutineScope.launch {
    experienceRenderer.onViewConfigurationChanged(renderContext)
}
```

right here?

Reply:
it is important it goes through the viewModel because the viewModel is the one getting nulled when the experience is dismissed/removed from screen permanently. probably wont make much of difference but this way we ensure that we are only reporting configuration changed when we should for a given renderContext.

Reply:
ah ok, so I guess it could null check `viewModel` here? or really it wont matter since the `Reset` action would do nothing if it was in Idling state? but this current implementation seems fine too - I was just thinking it may possibly be able to be simplified. I don't think it really needs to change though if this is most clear.